<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(0, 0, 0);
}

QPushButton{
background-color: #000000ff;
border-radius:20px;
}

QPushButton:hover{
background-color: rgb(53,53,79);
border-radius:20px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(20, 200, 220);</string>
            </property>
            <property name="text">
             <string>Menu Juegos</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>391</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_minimizar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Svg/Minimizar.png</normaloff>Svg/Minimizar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_restaurar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Svg/Minimo.png</normaloff>Svg/Minimo.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_maximizar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Svg/Expandir.png</normaloff>Svg/Expandir.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_cerrar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Svg/Cerrar.png</normaloff>Svg/Cerrar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="menu_lateral">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: rgb(53, 53, 79);
}

QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20, 200, 220);
border-radius: 5px;
border: 1px solid white;
}

QPushButton:hover{
background-color:black;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="bt_Numero">
               <property name="text">
                <string> Adivina el numero</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>Svg/Number.png</normaloff>Svg/Number.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>64</width>
                 <height>64</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="styleSheet">
                <string notr="true">QFrame{
background-color: rgb(53, 53, 79);
}

QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20, 200, 220);
border-radius: 5px;
border: 1px solid #14C8DC;
}

QPushButton:hover{
background-color: black;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>4</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>100</number>
                </property>
                <item>
                 <widget class="QPushButton" name="bt_menu_uno">
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>Svg/izquierda.png</normaloff>Svg/izquierda.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bt_menu_dos">
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>Svg/Derecha.png</normaloff>Svg/Derecha.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>32</width>
                    <height>32</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_python">
               <property name="maximumSize">
                <size>
                 <width>15000</width>
                 <height>15000</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QFrame{
background-color: rgb(53, 53, 79);
}

QLabel{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20, 200, 220);
border: 0px solid #14C8DC;
}

QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20, 200, 220);
border-radius: 5px;
border: 1px solid #14C8DC;
}
QLineEdit{
border: 2px solid #14C8DC;
border-radius: 8px;
color: rgb(255, 255, 255);
background-color: rgb(0,0,0);
font: 75 12pt &quot;Times New Roman&quot;;
}

QPushButton:hover{
background-color: black;
}</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="Etiqueta">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="entrada_nombre">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="line">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="Etiqueta_2">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="entrada_numero"/>
                </item>
                <item>
                 <widget class="Line" name="line_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="boton_verificar">
                  <property name="text">
                   <string>Verificar</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color: rgb(53, 53, 79);
}

QLabel{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20, 200, 220);
border: 0px solid #14C8DC;
}

QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20, 200, 220);
border-radius: 5px;
border: 1px solid #14C8DC;
}
QLineEdit{
border: 2px solid #14C8DC;
border-radius: 8px;
color: rgb(255, 255, 255);
background-color: rgb(0,0,0);
font: 75 12pt &quot;Times New Roman&quot;;
}

QPushButton:hover{
background-color: black;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="etiqueta_resultado">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
